iroh-relay/src/server/http_server.rs

```diff
@@ -539,7 +539,7 @@ impl RelayService {
                     Err(err) => warn!("upgrade error: {err:#}"),
                 }
             }
-            .instrument(debug_span!("handler"))
+            .instrument(warn_span!("handler"))
```

Comment:
Not that this does any harm, but a span without fields is perhaps not that interesting?

Reply:
potentially, but we have quite a few of these currently